WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -196,9 +196,11 @@ final class ProductFormViewController<ViewModel: ProductFormViewModelProtocol>:
         view.endEditing(true)
 
         if isBeingDismissedInAnyWay {
-            productImageUploader.startEmittingErrors(key: .init(siteID: viewModel.productModel.siteID,
-                                                                productOrVariationID: productOrVariationID,
-                                                                isLocalID: !viewModel.productModel.existsRemotely))
+            let key = ProductImageUploaderKey(siteID: viewModel.productModel.siteID,
+                                              productOrVariationID: productOrVariationID,
+                                              isLocalID: !viewModel.productModel.existsRemotely)
+            productImageUploader.startEmittingErrors(key: key)
+            productImageUploader.sendBackgroundUploadNoticeIfNeeded(key: key, using: ServiceLocator.noticePresenter)
```

Comment:
Opinion: Moving the product image related code to a helper method will improve readability.

Reply:
Done in 700d88f9ef57a46349b1ec70c021897bcc3b9323.